CIPs/0012.md

```diff
@@ -0,0 +1,37 @@
+# CIP 0012: More granular validator signer keys
+
+- Date: 2020-05-20
+- Author: @kobigurk
+- Status: WIP
+
+## Overview
+
+Validator signer keys allow to both sign consensus blocks and rotate BLS keys. We suggest to add more granularity to support the following use case: ECDSA signing of consensus messages without the ability to rotate BLS keys. This would be useful for situations where the BLS key is well protected while the ECDSA key isn't. Since the BLS key signs the COMMIT messages, the danger rests mostly there anyway. Examples of these kinds of situations are threshold signatures for BLS and Ledger signing for BLS.
```

Comment:
What are the situations in which the BLS key is better protected than the ECDSA key?

Reply:
Expanded the description to answer it :)